src/System.Security.Cryptography.Encoding/tests/Configurations.props

```diff
@@ -1,8 +1,8 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Unix;
-      netstandard-Windows_NT;
+      netcoreapp;
```

Comment:
I think all the conditional compilations can be removed and this can go back to just `netstandard`.  (Not sure why it was RID-split before)

Reply:
Keeping the RID split to reduce the number of changes even though it's not necessary.